meta-mender-intel/scripts/manifest-intel.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+  <manifest>
+  <default sync-j="4" revision="sumo"/>
+
+  <remote fetch="git://git.yoctoproject.org"        name="yocto"/>
+  <remote fetch="git://github.com/linux-sunxi"      name="sunxi"/>
+  <remote fetch="git://git.openembedded.org"        name="oe"/>
```

Comment:
Why do we need openembedded? Isn't yoctoproject enough?

Reply:
Also a left over from copy/paste. Poky is enough and meta-intel does not have any other dependencies.